.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,41 @@
+# .github/workflows/chromatic.yml
+
+name: 'Chromatic'
+
+on: push
```

Comment:
on push to all branches? In the past, we've done visual regression testing on pr 🤔

Reply:
not sure what the intended usecase is based on the ticket but this is good!

Reply:
cc @matthewroach

Reply:
The action should be running on all pushes - it's the best way for chromatic to keep track of everything, and you need to run on head branches. Arc Theme Blocks is a good example - https://github.com/WPMedia/arc-themes-blocks/blob/arc-themes-release-version-1.19/.github/workflows/chromatic.yml